recipes/bibmon/LICENSE.md

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This file is in the tarball. No need to add it here.

Reply:
Ok

Reply:
Please mark as resolved when it is resolved are we will lose track of things.